### ISSUE TITLE
Yakouchuu II - Satsujun Kouru: HVQM - wrong task (type 1 instead 7)

### ISSUE THREAD

**Issue by olivieryuyu** [project64/project64#1567]
https://github.com/mupen64plus/mupen64plus-core/issues/546

See discussion and potential fix here.

**Comment by StenApp** [comment]
Is it yet implemented?

**Comment by StenApp** [comment]
The implemented it.
https://github.com/mupen64plus/mupen64plus-rsp-hle/pull/69

**Comment by CrashOveride95** [comment]
not complete yet I thought?

**Comment by StenApp** [comment]
> 

> not complete yet I thought?

Yes, but work on a solution. Does it work with project64? How do they solve it?
